mllib/src/main/scala/org/apache/spark/ml/util/ReadWrite.scala

```diff
@@ -446,6 +463,20 @@ private[ml] object DefaultParamsReader {
     val cls = Utils.classForName(metadata.className)
     cls.getMethod("read").invoke(null).asInstanceOf[MLReader[T]].load(path)
   }
+
+  def loadInitialModel[M <: Model[M]](instance: Params, path: String, sc: SparkContext): Unit = {
```

Comment:
Also, we should call it `loadAndSetInitialModel` to be clear about side effects.